docs/resources/User.md

```diff
@@ -179,11 +179,14 @@ Create a new DM channel with a user. Returns a [DM channel](#DOCS_RESOURCES_CHAN
 
 ## Create Group DM % POST /users/@me/channels
 
-Create a new group DM channel with multiple users. Returns a [DM channel](#DOCS_RESOURCES_CHANNEL/channel-object) object. This endpoint was intended to be used with the now-deprecated GameBridge SDK. DMs created with this endpoint will not be shown in the Discord client
+Create a new group DM channel with multiple users. Returns a [DM channel](#DOCS_RESOURCES_CHANNEL/channel-object) object. This endpoint was intended to be used with the now-deprecated GameBridge SDK.
 
 > warn
 > This endpoint is limited to 10 active group DMs.
 
+> info
```

Comment:
IIRC, there are issues with Discord's documentation compiler when successive notice boxes. You may need to merge this with the above warning. I suggest keeping the `info` style, if you aren't sure which one should be kept.

Reply:
@z64 Seems to be fine [here](https://discord.com/developers/docs/resources/guild#list-guild-members)

Reply:
Hm, alright. Wasn't aware we had any cases of it still, must have been fixed at some point.. thanks.